build.sh

```diff
@@ -4,7 +4,7 @@ rm -rf build
 mkdir build
 cd build
 cmake ..
-make
+make -j$(nproc)
```

Comment:
Does it work with docker? in my laptop it was failing with out of memory exceptions. I wouldn't change this script since it is used by the build machine in jenkins

Reply:
removed